src/components/Status/stylesBuilder/index.ts

```diff
@@ -0,0 +1 @@
+export { stylesBuilder } from './stylesBuilder';
```

Comment:
I know that there is a lot of cases that stylesBuilider is in the seperate folder, but I don't see a reason folder named `stylesBuilder` with `index.ts` and `stylesBuilder.ts` inside would be better than just a file `stylesBuilder.ts`. @adrian-potepa wdyt?

Reply:
I agree with @mwleklinskiVL - if we have one file or there is no need to export more complex bundles of functions, it no need to add `index.ts` to export ;)

Reply:
Isn't it neater current way? In my opinion component file (in this case `Status.tsx`)should have easily findable content so moving `stylesBuilder` function back would result in being more messy (rendered content wouldn't be so easy to find).
And about barrel file `index.ts` - that was a pattern I used to use because it makes imports cleaner. I found out that using this pattern is a subject of discussion. It makes a package bigger -  [Barrel files and why you should STOP using them now](https://dev.to/tassiofront/barrel-files-and-why-you-should-stop-using-them-now-bc4) so I'll go with it and delete `index.ts` file

Reply:
I don't quite understand how it is easier to find `stylesBuilder.ts` file inside a folder stylesBuilder then without that folder, and it results in imports ending with `../stylesBuilder/stylesBuilder`

Reply:
I meant `Status.tsx` file - that it's more readable. You see the content which is rendered instead of funcions and other things that don't need to be in the file. But it's just a suggestion based on some articles about clean code in React that suggest that component files should be dumb and responsible just for rendering UI. 

I can move it to component file but to be consistent I think we should move all `stylesBuilder`s to the component files.

Reply:
Yes, the Status.tsx file should stay as it is, and with the `stylesBuilder,` we wanted to make a rule only to create the folder `stylesBuilder` if there is some complex logic and we want to create more files with helper functions for most of the situations this folder is not necessary (so single `stylesBuilder.ts` file would suffice)

Reply:
Ok, so what do you think about other cases like that? In my opinion, we should move `stylesBuildre.ts` to the other components folder as well. So when the next developers come they don't follow the same pattern I did.

Well I see most (or all) of them are the same case

Reply:
moved `stylesBuilder.ts`